### PR TITLE
CSS Data Types: Add calculation data types `<calc-sum>`, `<calc-product>`, `<calc-value>` and `<calc-constant>`

### PR DESCRIPTION
**Pull request by ramiy**
### Description

Document CSS calculation data types.

### Related issues and pull requests

https://github.com/mdn/mdn/issues/242

### CODE REVIEW

File: files/en-us/web/css/css_types/index.md [hunk 1]
```diff
@@ -133,6 +133,19 @@ The {{cssxref("&lt;position&gt;")}} data type is interpreted as defined for the
...
+## Calculation data types
+
+These data types are used in [CSS math function](/en-US/docs/Web/CSS/CSS_Functions#math_functions) calculations.
+
+- {{cssxref("&lt;calc-sum&gt;")}}
+  - : A calculation which is a sequence of calculation values interspersed with addition (`+`) and subtraction (`-`) operators. This data type requires both values to have units.
+- {{cssxref("&lt;calc-product&gt;")}}
+  - : A calculation which is a sequence of calculation values interspersed with multiplication (`*`) and division (`/`) operators. When multiplying, one value must be unitless. When dividing, the second value must be unitless.
+- {{cssxref("&lt;calc-value&gt;")}}
+  - : Defines accepted values fo calculations, values such as {{cssxref("&lt;number&gt;")}}, {{cssxref("&lt;dimension&gt;")}}, {{cssxref("&lt;percentage&gt;")}}, {{cssxref("&lt;calc-constant&gt;")}} or nested {{cssxref("&lt;calc-sum&gt;")}} calculations.
```
Comment (wbamberg) [inline files/en-us/web/css/css_types/index.md:145]:
```suggestion
  - : Defines accepted values for calculations, values such as {{cssxref("&lt;number&gt;")}}, {{cssxref("&lt;dimension&gt;")}}, {{cssxref("&lt;percentage&gt;")}}, {{cssxref("&lt;calc-constant&gt;")}} or nested {{cssxref("&lt;calc-sum&gt;")}} calculations.
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @ramiy !

**Comment by schalkneethling** [comment]
Whoops, this one should perhaps now have been merged just yet as there was still an open question here: https://github.com/mdn/mdn/issues/242

**Comment by ramiy** [comment]
@schalkneethling I answered the question. No need to revert the merge.

**Pull request merged**